easy-ci.php

```diff
@@ -94,5 +95,7 @@
         NamespacedNameDecorator::class,
         NamedVariableFactory::class,
         BinaryOpTreeRootLocator::class,
+
+        NodeVisitorInterface::class,
```

Comment:
may I ask what this skip-list is about and when a class needs to be added here?

Reply:
without this, class implements NodeVisitorInterface will be marked as unused